.github/workflows/build.yml

```diff
@@ -11,13 +11,15 @@ jobs:
       - uses: actions/setup-java@v3
         with:
           distribution: temurin
-          java-version: 11
+          java-version: 17
```

Comment:
I would do this in a separate PR if needed 
It makes sense to find out what happened with setup-android github actions (it worked before) 
Probably it sets up the latest SDK when no specific version is specified

Reply:
Yep, we don't need it, it's just to get CI running.

Reply:
Actually, by switching to boringssl instead of bouncy castle or jdk I was able to stay on Java 11.